modules/components/pages/inputs/redpanda.adoc

```diff
@@ -0,0 +1,619 @@
+
+*Default*: `[]`
+
+```yml
+# Examples
+
+client_certs:
+  - cert: foo
+    key: bar
+
+client_certs:
+  - cert_file: ./example.pem
+    key_file: ./example.key
+```
+
+=== `tls.client_certs[].cert`
+
+The plain text certificate to use.
+
+*Type*: `string`
+
+*Default*: `""`
+
+=== `tls.client_certs[].key`
+
+The plain text certificate key to use.
+
+include::components:partial$secret_warning.adoc[]
+
+*Type*: `string`
+
+*Default*: `""`
+
+=== `tls.client_certs[].cert_file`
+
+The path to the certificate to use.
+
+*Type*: `string`
+
+*Default*: `""`
+
+=== `tls.client_certs[].key_file`
+
+The path of a certificate key to use.
+
+*Type*: `string`
+
+*Default*: `""`
+
+=== `tls.client_certs[].password`
+
+The plain text password for when the private key is password encrypted in PKCS#1 or PKCS#8 format. The obsolete `pbeWithMD5AndDES-CBC` algorithm is not supported for the PKCS#8 format. 
+
+WARNING: The `pbeWithMD5AndDES-CBC` algorithm does not authenticate ciphertext, and is vulnerable to padding Oracle attacks which may allow an attacker recover the plain text password.
+
+include::components:partial$secret_warning.adoc[]
+
+*Type*: `string`
+
+*Default*: `""`
+
+```yml
+# Examples
+
+password: foo
+
+password: ${KEY_PASSWORD}
+```
+
+=== `sasl`
+
+Specify one or more methods or mechanisms of SASL authentication. They are tried in order. If the broker supports the first SASL mechanism, all connections use it. If the first mechanism fails, the client picks the first supported mechanism. If the broker does not support any client mechanisms, all connections fail.
+
+*Type*: `array`
+
+
+```yml
+# Examples
+
+sasl:
+  - mechanism: SCRAM-SHA-512
+    password: bar
+    username: foo
+```
+
+=== `sasl[].mechanism`
+
+The SASL mechanism to use.
+
+
+*Type*: `string`
+
+
+|===
+| Option | Summary
+
+| `AWS_MSK_IAM`
+| AWS IAM-based authentication as specified by the `aws-msk-iam-auth` Java library
+| `OAUTHBEARER`
+| OAuth Bearer-based authentication
+| `PLAIN`
+| Plain text authentication
+| `SCRAM-SHA-256`
+| SCRAM-based authentication as specified in RFC 5802
+| `SCRAM-SHA-512`
+| SCRAM-based authentication as specified in RFC 5802
+| `none`
+| Disable SASL authentication
+
+|===
+
+=== `sasl[].username`
+
+A username for `PLAIN` or `SCRAM-*` authentication.
+
+*Type*: `string`
+
+*Default*: `""`
+
+=== `sasl[].password`
+
+A password for `PLAIN` or `SCRAM-*` authentication.
+
+include::components:partial$secret_warning.adoc[]
+
+*Type*: `string`
+
+*Default*: `""`
+
+=== `sasl[].token`
+
+The token to use for a single session's `OAUTHBEARER` authentication.
+
+*Type*: `string`
+
+*Default*: `""`
+
+=== `sasl[].extensions`
+
+Key/value pairs to add to `OAUTHBEARER` authentication requests.
+
+*Type*: `object`
+
+
+=== `sasl[].aws`
+
+AWS specific fields for when the `mechanism` is set to `AWS_MSK_IAM`.
+
+*Type*: `object`
+
+
+=== `sasl[].aws.region`
+
+The AWS region to target.
+
+
+*Type*: `string`
+
+*Default*: `""`
+
+=== `sasl[].aws.endpoint`
```

Comment:
Are all AWS fields configurable in Cloud?

Reply:
Yes